test/images/sample-device-plugin/sampledeviceplugin.go

```diff
@@ -104,6 +115,15 @@ func main() {
 
 	}
 	dp1.SetAllocFunc(stubAllocFunc)
+
+	cdi := os.Getenv("CDI_ENABLED")
```

Comment:
Let's call this `cdiEnabled` to be more explicit. We can then use:
```suggestion
	cdiEnabled := os.Getenv("CDI_ENABLED") != ""
```